app/views/activity_mailer/activity_notification.text.erb

```diff
@@ -1,6 +1,6 @@
 <%= @activity %>
 
-<%= t("mailer.proposal_link_text",
-      proposal_url: proposal_url(@proposal)) %>
+<%= t("mailer.view_request_cta") %>
+<%= proposal_url(@proposal) %>
```

Comment:
this was how the rest of the `cta` links were done so I wanted to stick with that for consistency. not opposed to moving all to passing link as a param just went with what was happening more often